test/image/mocks/multicategory_series.json

```diff
@@ -0,0 +1,1617 @@
+{
```

Comment:
There is a bug in current image test system. 
To avoid that please rename this new mock to start with `z` e.g. zz-multicategory_series.json